dashboard/views.py

```diff
@@ -46,13 +46,7 @@ def get(self, request, *args, **kargs):  # pylint: disable=unused-argument, no-s
         certificates = edx_client.certificates.get_student_certificates(
             request.user.username, enrollments.get_enrolled_course_ids())
 
-        response_data = {'courses': []}
+        response_data = []
         for program in Program.objects.all():
-            for course in program.course_set.all():
-                response_data['courses'].append(
-                    get_info_for_course(
-                        request.user, course, enrollments, user_certificates=certificates)
-                )
-
-        response_data['courses'].sort(key=lambda x: x['position_in_program'])
```

Comment:
I asked this question yesterday to @pdpinch and he replied something like "probably, but let's postpone this"